contracts/LpSugar.vy

```diff
@@ -634,7 +636,7 @@ def positions(_limit: uint256, _offset: uint256, _account: address)\
 
     if self._is_cl_factory(factory.address):
       # fetch unstaked CL positions
-      positions_count: uint256 = self.nfpm.balanceOf(_account)
+      positions_count: uint256 = nfpm.balanceOf(_account)
```

Comment:
I think we can remove the `_nfpm` and instead fetch it off the first factory pool. This should keep the existing interface. 

Probably easier if we swap the fetching of staked positions to be the first loop (line 665) and cache the `nfpm` around line 678

Wdyt?

Reply:
@pedrovalido are we still doing this?